components/homme/src/prim_main.F90

```diff
@@ -240,6 +240,11 @@ end subroutine finalize_kokkos_f90
      
      nstep = nextoutputstep(tl)
      do while(tl%nstep<nstep)
+
+     if(tl%nstep < 2) then 
+        call t_disablef()
```

Comment:
Does this change have implications for E3SM more widely? My understanding of `t_disablef` is it would affect timer output in all E3SM runs.

Reply:
Never mind: I see now this is in `program main` for standalone Homme. It's still worth double-checking if you really want this chang.e

Reply:
yes, i do. the issue is that even with aot build, timers are polluted because some TPL libs are built with jit. this change resolved the issue.